public/app/core/specs/file_export.test.ts

```diff
@@ -53,10 +53,10 @@ describe('file_export', () => {
         '"Time";"series_1";"series_2"\r\n' +
         '"1500026100";1;11\r\n' +
         '"1500026200";2;12\r\n' +
-        '"1500026300";null;13\r\n' +
-        '"1500026400";null;null\r\n' +
-        '"1500026500";null;15\r\n' +
-        '"1500026600";6;null';
+        '"1500026300";;13\r\n' +
```

Comment:
this looks wrong, no longer outputting null

Reply:
I mentioned this change in the description and in the commit message.
And I leaved [a corresponding comment](https://github.com/grafana/grafana/pull/18179/files#diff-9d9218dee66e94b7d96fa5d7cb19ab83R57) in the `formatRow` as well.

The idea here is to keep the resulting CSV readable and exclude meaningless values.

Reply:
Currently, without these changes, null values are rendered as `"'-"` which is a garbage.

Reply:
They should be rendered as null so it’s clear it’s null

Reply:
> They should be rendered as null so it’s clear it’s null

This is not how the export works now because all values are currently formatted regardless of their types.

Anyway, is there any good reason to render `null` values as `null` strings in CSV instead of keeping corresponding cells empty?
CSV in this case looks strange or even broken for any end user who is not a JS/TS programmer.

Reply:
Let me repeat my question.
> is there any good reason to render `null` values as `null` strings in CSV instead of keeping corresponding cells empty?